okhttp/src/main/java/io/grpc/okhttp/OkHttpClientStream.java

```diff
@@ -210,6 +233,7 @@ public void cancel(Status reason) {
     /** True iff neither {@link #cancel} nor {@link #start(int)} have been called. */
     @GuardedBy("lock")
     private boolean canStart = true;
+    private Tag tag;
```

Comment:
final?

Reply:
Done. You may also want that in Netty's. 😜